spec/functions/number.md

```diff
@@ -655,9 +655,13 @@ Implementations MUST NOT substitute the unit without performing the associated c
 ### Number Operands
 
 The _operand_ of a number function is either an implementation-defined type or
-a literal whose contents match the `number-literal` production in the [ABNF](/spec/message.abnf).
+a literal whose contents match the following `number-literal` production.
 All other values produce a _Bad Operand_ error.
 
+```abnf
+number-literal = ["-"] (%x30 / (%x31-39 *DIGIT)) ["." 1*DIGIT]
```

Comment:
This is different from the `number-literal` in the current ABNF and inconsistent with @gibson042's comment (and my reply to it) in that it leaves out the exponent part. If we leave out exponents in this ABNF, we should probably permit implementations to support them. Given that the original ABNF came from JSON, is there any reason to deviate from it?

Reply:
I agree, we should keep the optional exponent here.

Reply:
Done.